base_transaction_id/models/invoice.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2011-2012 Nicolas Bessi (Camptocamp)
+# Copyright 2012-2015 Yannick Vaucher (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    transaction_id = fields.Char(string='Transaction ID',
```

Comment:
same remark about code formatting

Reply:
While I also prefer this format with return after opening parethesis I prefer not to change this to not alter too much the code base history.